app/src/main/java/com/example/composetemplate/presentation/screens/entry_screens/AuthenticationScreen.kt

```diff
@@ -7,7 +7,7 @@ import com.example.composetemplate.navigation.Navigator
 @Composable
 fun AuthenticationScreen(args: EntryScreens.Authentication, navigator: Navigator) {
     if (args.isAuthorized) {
-        LogInScreen(navigator, args.shouldShowAdd)
+        RegisterScreen(navigator, args.shouldShowAdd) //todo: move
```

Comment:
Can you explain this? where this component should move?

Reply:
It will change in the navigation mission before master